// [[Reason: Invalid computed property can only be detected by TSC]] ////

//// [tests/cases/conformance/classes/propertyMemberDeclarations/staticPropertyNameConflicts.ts] ////

===================================================================
--- TSC	declarations
+++ DTE	declarations
@@ -72,9 +72,8 @@
 staticPropertyNameConflicts.ts(272,5): error TS9008: Method must have an explicit return type annotation with --isolatedDeclarations
 staticPropertyNameConflicts.ts(277,12): error TS1319: A default export can only be used in an ECMAScript-style module.
 staticPropertyNameConflicts.ts(278,16): error TS2699: Static property 'prototype' conflicts with built-in property 'Function.prototype' of constructor function 'StaticPrototype'.
 staticPropertyNameConflicts.ts(284,12): error TS2699: Static property 'prototype' conflicts with built-in property 'Function.prototype' of constructor function 'ExportedStaticPrototype'.
-staticPropertyNameConflicts.ts(284,12): error TS9014: Computed properties must be number or string literals, variables or dotted expressions with --isolatedDeclarations
 staticPropertyNameConflicts.ts(289,12): error TS1319: A default export can only be used in an ECMAScript-style module.
 staticPropertyNameConflicts.ts(290,16): error TS2300: Duplicate identifier 'prototype'.
 staticPropertyNameConflicts.ts(290,16): error TS2699: Static property 'prototype' conflicts with built-in property 'Function.prototype' of constructor function 'StaticPrototypeFn'.
 staticPropertyNameConflicts.ts(296,12): error TS2300: Duplicate identifier '[FunctionPropertyNames.prototype]'.
@@ -90,9 +89,9 @@
 staticPropertyNameConflicts.ts(346,12): error TS9008: Method must have an explicit return type annotation with --isolatedDeclarations
 staticPropertyNameConflicts.ts(347,5): error TS9008: Method must have an explicit return type annotation with --isolatedDeclarations
 
 
-==== staticPropertyNameConflicts.ts (37 errors) ====
+==== staticPropertyNameConflicts.ts (36 errors) ====
     const FunctionPropertyNames = {
         name: 'name',
         length: 'length',
         prototype: 'prototype',
@@ -421,10 +420,8 @@
     export class ExportedStaticPrototype {
         static [FunctionPropertyNames.prototype]: number; // always an error
                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 !!! error TS2699: Static property 'prototype' conflicts with built-in property 'Function.prototype' of constructor function 'ExportedStaticPrototype'.
-               ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-!!! error TS9014: Computed properties must be number or string literals, variables or dotted expressions with --isolatedDeclarations
         [FunctionPropertyNames.prototype]: string; // ok
     }
     
     module TestOnDefaultExportedClass_6 {
